src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -138,6 +138,39 @@ export function transformDisabilities(disabilities = []) {
   );
 }
 
+export function transformMVPData(formData) {
```

Comment:
LGTM to me and satisfies the transformation requirements in the associated issue
- Remove `veteran` and put it's properties as root-level properties
- Nest `servicePeriods` under `serviceInformation`
- Nest `reservesNationalGuardService` under `serviceInformation`